routing/dht/dht_net.go

```diff
@@ -227,3 +228,23 @@ func (ms *messageSender) SendRequest(ctx context.Context, pmes *pb.Message) (*pb
 
 	return mes, nil
 }
+
+func (ms *messageSender) ctxReadMsg(ctx context.Context, mes *pb.Message) error {
+	t := time.NewTimer(time.Second * 30)
```

Comment:
I honestly think we could get away without the timeout here and just stick to respecting the context and fixing utp. 

What do you think?